advanced/Scripts/update.sh

```diff
@@ -104,34 +102,40 @@ echo ":::"
 
 
 
-if [[ ${piholeVersion} == ${piholeVersionLatest} && ${webVersion} == ${webVersionLatest} ]]; then
+if [[ "${piholeVersion}" == "${piholeVersionLatest}" ]] && [[ "${webVersion}" == "${webVersionLatest}" ]]; then
 	echo "::: Everything is up to date!"
 	echo ""
 	exit 0
 
-elif [[ ${piholeVersion} == ${piholeVersionLatest} && ${webVersion} != ${webVersionLatest} ]]; then
+elif [[ "${piholeVersion} == ${piholeVersionLatest}" ]] && [[ "${webVersion}" != "${webVersionLatest}" ]]; then
 	echo "::: Pi-hole Web Admin files out of date"
-	getGitFiles ${webInterfaceDir} ${webInterfaceGitUrl}
+	getGitFiles "${WEBINTERFACEDIR}" "${WEBINTERFACEGITURL}"
 	echo ":::"
-	webVersion=$(pihole -v -a -c)
+	webVersion=$(/usr/local/bin/pihole -v -a -c)
 	echo "::: Web Admin version is now at ${webVersion}"
 	echo "::: If you had made any changes in '/var/www/html/admin', they have been stashed using 'git stash'"
```

Comment:
Good for first pass, future improvement maybe detect if modifications have been made and only show then and run stash if actually necessary/forced to because of a dirty repo status.

Is stashing going to fail if you stashed modifications 2 updates in a row?